BFR0QN/ReadJsonFile.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text.Json;
+
+namespace BFR0QN
+{
+    public static class ReadJsonFile
+    {
+        public static List<Food> ReadJsonFileToList(string jsonFileName)
+        {
+            List<Food> foods = new List<Food>();
+            try
+            {
+                string jsonText = File.ReadAllText(jsonFileName);
+                foods = JsonSerializer.Deserialize<List<Food>>(jsonText,new JsonSerializerOptions(JsonSerializerDefaults.Web));
```

Comment:
Miért Lista? Ez lehetne tömb is, btw, mert adott mennyi elem van a JSON fájlban.